woocommerce/woocommerce-ios
# Update App Store Metadata

This PR is part of merging https://github.com/woocommerce/woocommerce-ios/pull/3337.

It looks like we haven't previously synced metadata from App Store Connect into the app repo. Additionally, some of the locale codes were incorrect.

This PR fixes those inconsistencies by:

- Ensuring that we're using the correct locale codes for App Store Connect
- Ensuring that when data is downloaded from GlotPress, it's moved into the correct directory
- Ensuring that all the keys we need for screenshots are always downloaded

**To Test:**
- Check out this PR
- Delete everything in the `fastlane/metadata` directory (but not the directory itself)
- Run `bundle exec fastlane deliver download_metadata`, then run `bundle exec fastlane download_app_store_translations`. The repo should now be clean.
